language/bert/.ipynb_checkpoints/squad_QSL-checkpoint.py

```diff
@@ -0,0 +1,137 @@
+# coding=utf-8
```

Comment:
.pynb files need not be uploaded right?

Reply:
I am not too sure why the notebook checkpoint files are up. I think they are not necessary.